lib/components/map/elevation-point-marker.tsx

```diff
@@ -1,18 +1,33 @@
+import { connect } from 'react-redux'
+import { Leg } from '@opentripplanner/types'
+import { Marker } from 'react-map-gl'
 import coreUtils from '@opentripplanner/core-utils'
 import React, { Component } from 'react'
-import { connect } from 'react-redux'
-import { CircleMarker } from 'react-leaflet'
 
+type Props = {
+  diagramLeg: Leg
+  elevationPoint: number
+  showElevationProfile?: boolean
+}
 /**
  * As the OTP user moves the cursor over the elevation tracking chart
  * of a walking or biking leg (to see which point of their itinerary is at which elevation),
  * ElevationPointMarker displays and moves a marker on the map to highlight
  * the location that corresponds to the cursor position on the elevation chart,
  * so the user can see the streets and paths that correspond to a portion of an elevation profile.
  */
-class ElevationPointMarker extends Component {
-  render () {
+class ElevationPointMarker extends Component<Props> {
```

Comment:
Convert to a function component.

Reply:
Converted in d3575aa5c7d8ea8b45d8959c9d74f64ac03b0274